### PR TITLE
Add page for no permission

### PR DESCRIPTION
**Pull request by insacuri**
# What and why?

# How to test?

# Trello

### CODE REVIEW

File: response_operations_ui/views/errors.py [hunk 1]
```diff
@@ -45,6 +46,12 @@ def handle_authentication_error(error):
     return redirect(url_for("sign_in_bp.sign_in"))
 
 
+@error_bp.app_errorhandler(NoPermissionError)
```
Comment (LJBabbage) [inline response_operations_ui/views/errors.py:49]: :thumbsup: We need to do this more often

File: response_operations_ui/views/messages.py [hunk 2]
```diff
@@ -53,8 +56,8 @@
 @login_required
 def create_message():
     if not user_has_permission("messages.edit"):
-        logger.exception("No message edit role")
-        abort(401)
+        logger.error("No message edit role")
```
Comment (LJBabbage) [inline response_operations_ui/views/messages.py:59]: Without any biddings (i.e user_id) and the volume of log message we get do these add anything? With it raising to NoPermissionError which also has an error do we need them?

**Review by LJBabbage — APPROVED** [review]
Created a new secure message with admin account, when I tried to access the close page with uaa_reporting_units_edit_user@ons.gov.uk it gave me the new message. Minor comment about logging

File: response_operations_ui/views/messages.py (hunk 2 above)
Comment (insacuri) [reply response_operations_ui/views/messages.py:59]: Agreed.  Just changed it so we add the missing permission to the exception, and the NoPermissionError handler logs both the users id and the missing permission

**Pull request merged**